modules/model/shared/src/main/scala/gem/data/Zipper.scala

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-package gpp.util
+package gem
```

Comment:
@swalker2m @tpolecat Any opinions about going back to `gem`?
I'm fine with this change